src/Services/FallbackSearchService.php

```diff
@@ -59,7 +53,7 @@ public function handleSearchQuery(Request $request, ExternalSearch $externalSear
      * @param ExternalSearch $externalSearch
      * @return array
      */
-    private function getSearchResults(Request $request, ExternalSearch $externalSearch)
+    public function getSearchResults(Request $request, ExternalSearch $externalSearch)
```

Comment:
If this is only public for tests, it should also be possible to mock this when you set it to `protected`. This should also be possible for `createResponseFromSearchResult`. I am not 100% certain but I think that these can still be mocked.

Reply:
It's necessary to call those methods in `SearchService::doNavigation` that's why I set it to public.